resolwe_bio/processes/support_processors/multiqc.py

```diff
@@ -499,9 +521,13 @@ def run(self, inputs, outputs):
 
             elif d.process.type == "data:expression:featurecounts:":
                 name = os.path.basename(d.output.counts_summary.path)
-                create_symlink(
-                    d.output.counts_summary.path, os.path.join(sample_dir, name)
+                sum_featurecounts_columns(
+                    summary_file=d.output.counts_summary.path, out_file=f"summed_{name}"
```

Comment:
Is the out file `f"summed_{name}"` placed in the sample dir?

Reply:
Fixed in the latest push.